main/src/sphexa/sphexa.cpp

```diff
@@ -43,6 +43,7 @@
 #endif
 
 #include "cstone/domain/domain.hpp"
+#include "init/wind_shock_init.hpp"
 #include "init/factory.hpp"
```

Comment:
Actually, I don't see why this include is needed as `init/factory.hpp` includes `init/wind_shock_init.hpp` if HDF5 support is enabled. If you include this here, I would expect it to break the non-HDF5 build.